src/AdminConsole/AdminConsole.csproj

```diff
@@ -55,5 +55,13 @@
   <Target Name="Run NPM Build" AfterTargets="PostBuildEvent">
     <Exec Command="npm install" />
     <Exec Command="npm run build" />
+    

```

Comment:
This will copy the necessary files from node_modules to our statically served assets folder for getting charts to work.